layers/+tools/translate/README.org

```diff
@@ -0,0 +1,64 @@
+#+TITLE: Translate Layer
+
+#+TAGS: layer|tool
+
+[[file:img/screen-record.gif]]
+
+* Table of Contents                     :TOC_5_gh:noexport:
+- [[#description][Description]]
+  - [[#features][Features:]]
+- [[#install][Install]]
+- [[#usage][Usage]]
+- [[#configuration][Configuration]]
+  - [[#language-pair][Language Pair]]
+  - [[#highlighting][Highlighting]]
+  - [[#read-only][Read-only]]
+  - [[#face][Face]]
+
+* Description
+This layer is designed for doing translation jobs.
+
+** Features:
+- Paragraph-level original buffer and translation
+  buffer cursor highlighting and synchronizing
+- Integrate word/paragraph online translation
+
+* Install
+To use this configuration layer, add it to your =~/.spacemacs=. You will need to
+add =translate= to the existing =dotspacemacs-configuration-layers= list in this file.
+
+* Usage
+It's quite simple to use this layer. Open the translation file you are currently working
+on, and call command =translate-open-original-file= or =translate-select-original-buffer=
+to set the original article for referring. Now you can just keep your cursor in your
+translation buffer. The cursor in the original buffer will keep syncing with your cursor
+moving.
```

Comment:
This sentence sounds a little awkward though it gets the point across. I would have written it as "The cursor in the original buffer will be kept in sync with the cursor in the translation buffer" or something like that.